exonum-java-binding/CHANGELOG.md

```diff
@@ -24,6 +24,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `--jvm-debug` command line argument that allows JDWP debugging of node. (#629)
 - Support of separated messages format Exonum v0.10. (#574)
 - `ListIndexProxy#stream` to enable stream processing of list elements. (#661)
+- Support of Time oracle.
+
+  To enable this service put `ejb_app_services.toml` file into EJB App's directory
```

Comment:
are you sure that this part should be here (in changelog)?

Reply:
probably no, we could reference documentation instead